Buildings/Air/Systems/SingleZone/VAV/ChillerDXHeatingEconomizerController.mo

```diff
@@ -6,166 +6,248 @@ model ChillerDXHeatingEconomizerController
   parameter Modelica.SIunits.Temperature TSupChi_nominal
     "Design value for chiller leaving water temperature";
   parameter Real minAirFlo(
-    min=0,
-    max=1,
-    unit="1") = 0.2
+    final min=0,
+    final max=1,
+    final unit="1")
     "Minimum airflow fraction of system"
     annotation(Dialog(group="Air design"));
-
-  parameter Modelica.SIunits.DimensionlessRatio minOAFra "Minimum outdoor air fraction of system"
+  parameter Modelica.SIunits.DimensionlessRatio minOAFra
+    "Minimum outdoor air fraction of system"
     annotation(Dialog(group="Air design"));
-
-  parameter Modelica.SIunits.Temperature TSetSupAir "Cooling supply air temperature setpoint"
+  parameter Modelica.SIunits.Temperature TSetSupAir
+    "Cooling supply air temperature setpoint"
     annotation(Dialog(group="Air design"));
-
-  parameter Real kHea(min=Modelica.Constants.small) = 2
-    "Gain of heating controller"
-    annotation(Dialog(group="Control gain"));
-
-  parameter Real kCoo(min=Modelica.Constants.small)=1
-    "Gain of controller for cooling valve"
-    annotation(Dialog(group="Control gain"));
-
-  parameter Real kFan(min=Modelica.Constants.small) = 0.5
-    "Gain of controller for fan"
-    annotation(Dialog(group="Control gain"));
-
-  parameter Real kEco(min=Modelica.Constants.small) = 4
-    "Gain of controller for economizer"
-    annotation(Dialog(group="Control gain"));
+  parameter Buildings.Controls.OBC.CDL.Types.SimpleController controllerTypeHea=
+    Buildings.Controls.OBC.CDL.Types.SimpleController.PI
+    "Type of controller"
+    annotation(Dialog(group="Heating coil signal"));
```

Comment:
For the parameter group name, I suggest to use `Setpoints`, `Heating coil control`, `Cooling coil control`, `Fan control`, and `Economizer control`.

Reply:
changed the group names as you suggested, also for g36 controller